### PR TITLE
Add promises extension to eslint

### PR DESCRIPTION
**Pull request by yjpa7145**

### CODE REVIEW

File: packages/api/endpoints/elasticsearch.js [hunk 1]
```diff
@@ -163,17 +163,22 @@ async function changeIndex(req, res) {
...
 
-  await esClient.indices.updateAliases({
-    body: {
-      actions: [
-        { remove: { index: currentIndex, alias: aliasName } },
-        { add: { index: newIndex, alias: aliasName } }
-      ]
-    }
-  }).then(() => {
+  try {
```
Comment (laurenfrederick) [inline packages/api/endpoints/elasticsearch.js:166]: Is this change fixing a linter error?

Comment (yjpa7145) [reply packages/api/endpoints/elasticsearch.js:166]: Yeah, I'm pretty sure it was addressing this one: https://github.com/xjamundx/eslint-plugin-promise/blob/master/docs/rules/always-return.md

File: packages/api/bin/Importer.js [hunk 2]
```diff
@@ -56,10 +57,12 @@ class Importer extends Transform {
    * @returns {undefined} undefined
    */
   _final(callback) {
-    Promise.all(this.promises).then(() => {
-      console.log(`Finished restoring ${this.count} records to DynamoDB`);
-      callback();
-    }).catch(callback);
+    return nodeify(this.waitForRestorationToFinish(), callback);
```
Comment (Jkovarik) [inline packages/api/bin/Importer.js:60]: I'm increasingly less happy with linters explicitly telling you to use a npm module instead.   Just a general comment, not particularly concerned here.

Comment (yjpa7145) [reply packages/api/bin/Importer.js:60]: In the case of the lodash warnings, I definitely agree. In cases like this, though, I'd rather err on the side of caution. Messing around with Promises and callbacks is error-prone. I'm happy to punt that complexity off to a package in this case.

Comment (Jkovarik) [reply packages/api/bin/Importer.js:60]: Agreed, this specific case seems fine.

**Pull request merged**